e2etest/bgptests/infra_setup.go

```diff
@@ -18,7 +18,7 @@ import (
 const (
 	multiHopNetwork = "multi-hop-net"
 	metalLBASN      = 64512
-	externalASN     = 64513
+	externalASN     = 4200000000
```

Comment:
@oribon @gclawes wdyt about this change?
I think that using a 2 bytes for ibgp and 4 bytes for external is enough to demonstrate that we support it, but would like to hear your thought as this is gonna impact all our e2es.

Reply:
seems good enough